### ISSUE TITLE
Storage backup gets killed by ActivityManager for excessive CPU use

### ISSUE THREAD

**Issue by grote** [seedvault-app/seedvault#271]
```
ActivityManager: Killing 19003:com.stevesoltys.seedvault/u10a89 (adj 900): excessive cpu 56240 during 300002 dur=750325 limit=10
```

source: https://cs.android.com/android/platform/superproject/+/android-11.0.0_r1:frameworks/base/services/core/java/com/android/server/am/ActivityManagerService.java;l=17722

The notification then just gets stuck and never moves or closes.

Signal has a similar issue for its own backups: https://github.com/signalapp/Signal-Android/issues/7617

**Comment by chirayudesai** [comment]
I first noticed this while doing 2 storage backups at the same time, one had large files, the other had lots of files.

However, after that I tried it one by one, and it did this again for one of those instances.

**Comment by chirayudesai** [comment]
I also suspect that perhaps this may have occurred in some cases with SeedVault backups in general. Unless you look at the log and happened to spot this when a backup got stuck (which may have happened from time to time), you wouldn't have been able to tell.

It also may not have happened with that given how it'd be a lot less intensive, but still, you never know.

**Comment by grote** [comment]
Our process state here does not seem to be [`PROCESS_STATE_FOREGROUND_SERVICE`](https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/app/ActivityManager.java;l=511;drc=4d9cc95b5255fa8b24c7d90cdfb5ee2b2a016dea) which should prevent us from getting killed.

**Comment by grote** [comment]
should get fixed by 0394f2213e70f43fafb6646deaa7f5e04075cea9
